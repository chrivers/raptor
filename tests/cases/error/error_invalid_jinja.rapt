# line 1
$ for x inx
# line 3
