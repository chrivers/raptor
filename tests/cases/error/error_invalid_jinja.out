error: unexpected identifier, expected in
 --> tests/cases/error/error_invalid_jinja.rapt:2:9
  |
1 | # line 1
2 | $ for x inx
  |         ^^^ syntax error: unexpected identifier, expected in (in tests/cases/error/error_invalid_jinja.rapt:2)
3 | # line 3
  |
